nixos/modules/misc/ids.nix

```diff
@@ -347,6 +347,7 @@ in
       #mailman = 316;  # removed 2019-08-30
       zigbee2mqtt = 317;
       # shadow = 318; # unused
+      unifi-video = 319;
```

Comment:
Tried using systemd DynamicUser, but this service requires a lot of setup in the state directory which I could not implement with only systemd's StateDirectory. I couldn't find a way to combine tmpfiles.d with StateDirectory, they seem to conflict with one another, so I'm opting for tmpfiles.d instead of StateDirectory, and so have to abandon DynamicUser in favor of statis uid/gid.

Reply:
Hmm, you still shouldn't need a static uid/gid; you can just let the system generate one for the user when it's added. See https://github.com/NixOS/rfcs/blob/master/rfcs/0052-dynamic-ids.md.

Reply:
Ah ok cool I wasn't aware of that mechanism, I was using existing modules as a reference which were using static uid/gid. I've now implemented this suggestion.